python2/httplib2/__init__.py

```diff
@@ -1958,6 +1983,7 @@ def request(
                             ca_certs=self.ca_certs,
                             disable_ssl_certificate_validation=self.disable_ssl_certificate_validation,
                             ssl_version=self.ssl_version,
+                            key_password = certs[0][2],
```

Comment:
Just this minor spaces around '=', CI linter should've caught this.

Reply:
Done